src/Hazelcast.Net/NearCaching/LfuComparer.cs

```diff
@@ -40,4 +40,19 @@ public int Compare(NearCacheEntry x, NearCacheEntry y)
             return cx.CompareTo(cy);
         }
     }
+
+    /// <summary>
+    /// Compares <see cref="NearCacheEntry"/> randomly.
+    /// </summary>
+    internal class RandomComparer : IComparer<NearCacheEntry>
```

Comment:
Is it okay to put this class under the `LfuComparer.cs` filename?

Reply:
Totally not, fixing